src/components/Scene.java

```diff
@@ -6,28 +6,80 @@
 import java.util.List;
 
 public class Scene extends Node {
-    public String sceneName;
+    public String name;
     public List<String> texts = new ArrayList<>();
     public List<Choice> choices = new ArrayList<>();
-    public int timer;
+    public int timer = 0;
 
     public String bgmFile;
     public String soundFile;
     public String pictureFile;
-    public String picturePosition;
+    public String picturePosition = "center";
 
     @Override
     public void parse() {
         // Check for scene name
+        name = tokenizer.getNext();
+
         // Check for texts
         // Check for choices
         // Check for timer
         // Check for bgm
         // Check for picture file; if picture exists, check for position
+        while (true) {
```

Comment:
Going to need to alter this. Kind of hard to tell where the next scene starts. @thethomasz @BoredX got any ideas?

Reply:
we can change our language so that there's an end? or we could do the same thing i did in my PR: https://github.com/mrbrendanwong/beta_engine/pull/5/files#diff-b2b041f1ffbd0f3baf3573dfd68a738aR46

Reply:
Game loops through scenes. Scene.parse should only create one scene per call

Reply:
Seems fine, we can go with that

Reply:
I think that wouldn't work for the last scene, because the next token would be DEATH SCENES or END SCENES, which are key words

Reply:
@wong-alan we were wrong and fake. this looks good